src/app.tsx

```diff
@@ -100,7 +100,7 @@ class App extends React.Component<{
         schemes = installedTheme.schemes;
         activeScheme = installedTheme.activeScheme;
       } else {
-        console.log("No theme set as installed");
+        // console.log("No theme set as installed");
```

Comment:
Can we change most of these logs to console.debug instead of commenting them out? They are useful for debugging, and changing to debug would hide them in the console by default.

Reply:
Ofcourse feel free to add `console.debug` where ever you want.